### ISSUE TITLE
Enhanced Auto Mine pause the time of the game, disabling any time-based aspects

### ISSUE THREAD

**Issue by wdos3** [Ephenia/Pokeclicker-Scripts#363]
I am still in Kanto at the moment, but I have noticed that it has stopped the following things from working:

Negative:
1. Any farm (time will not progress)
2. Pokemon Attack (clicks still deals dmg, but no Pokemon attack deals dmg)

Positive:
1. Items have an infinite duration (as long as Auto Mine script is enabled)
2. Gyms/Dungeons have an infinite duration

And while I am unsure as to whether the other scripts are broken as a result of Auto Mine, I must still mention the following scripts don't work:
1. Auto Safari
2. Infinite Event
3. Auto Quest
4. Discord Code Generator

**Comment by Meggidow** [comment]
I can tell that I've the same problem since yesterday morning, I can't even leave the game without saving my progression or it will rollback staight where I load the last time.

**Comment by ItsDeidara** [comment]
I'm noticing the same save game bug, and need to save manually. 

However I have automine working properly aloneside time based events. Do you have autosafari enabled by chance? From my troubleshooting it seems that, enabling that script itself may be causing the time bug since for me time only stops when that script is checked off.

**Comment by wdos3** [comment]
> I'm noticing the same save game bug, and need to save manually.
> 
> However I have automine working properly aloneside time based events. Do you have autosafari enabled by chance? From my troubleshooting it seems that, enabling that script itself may be causing the time bug since for me time only stops when that script is checked off.

Yeah, I have manually tested, and both auto-safari and auto-mine have done the whole "time stop" thing. The only difference is that auto-mine works, auto-safari doesn't. In addition, there are still many scripts that still work even with safari/mine active (like clicker, delay capture, oak items).
